tests/test_app_initializers.py

```diff
@@ -0,0 +1,33 @@
+import falcon
+import falcon.testing as testing
```

Comment:
A cosmetic nitpick, but this `import` form looks rather unorthodox; normally one uses `import as` to rename imported names. Maybe just `from falcon import testing`?